app/models/agreement_letter.rb

```diff
@@ -90,4 +95,13 @@ def too_big?(file)
     def wrong_filetype?(file)
       file.content_type != ALLOWED_MIMETYPE
     end
+
+  def wrong_file_trailer? file
```

Comment:
I don't think this function's name is descriptive of what it checks. Maybe `unable_to_open?`? Dunno, you'll probably have better ideas :wink: